package.json

```diff
@@ -11,7 +11,7 @@
     "test": "make test lint"
   },
   "dependencies": {
-    "sanctuary-type-classes": "4.0.x",
+    "sanctuary-type-classes": "5.0.x",
```

Comment:
This should happen in a separate pull request, before this pull request is merged.